### DISCUSSION TITLE
https://cucoders.dev/publicaciones/aprezcuba24/coffe-time/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#122] [Page Comments]
# https://cucoders.dev/publicaciones/aprezcuba24/coffe-time/

Trabajo en una oficina donde nos gusta tomar café, algo normal entre los programadores 😁. Pero como buen programador también nos gusta resolver problemas y automatizar procesos 😎. El problema era ¿Quién hace el café? 😈

https://cucoders.dev/publicaciones/aprezcuba24/coffe-time/

**Comment by bleach41** [comment]
un placer ser miembro de la casa del café 😁
